### DISCUSSION TITLE
compile librime on linux host

### DISCUSSION THREAD

**Discussion by ruijitek** [rime/home#1566] [Q&A] [answered]
Hello ,
    
      I am download https://github.com/rime/librime,then start make command,
that is process error below:

cmake . -Bbuild \
-DCMAKE_INSTALL_PREFIX=/usr \
-DCMAKE_BUILD_TYPE=Release \
-DBUILD_MERGED_PLUGINS=OFF \
-DENABLE_EXTERNAL_PLUGINS=ON
-- Boost version: 1.65.1
-- Found gflags: /usr/lib/x86_64-linux-gnu/libgflags.so
-- Found glog: /usr/lib/x86_64-linux-gnu/libglog.so
-- Found yaml-cpp: /usr/lib/x86_64-linux-gnu/libyaml-cpp.so
-- Found leveldb: /usr/lib/x86_64-linux-gnu/libleveldb.so
-- Found marisa: /usr/lib/x86_64-linux-gnu/libmarisa.so
CMake Error at cmake/FindOpencc.cmake:22 (message):
  Could not find opencc library.
Call Stack (most recent call first):
  CMakeLists.txt:132 (find_package)

-- Configuring incomplete, errors occurred!
See also "/data/reserve/input-methond/librime/build/CMakeFiles/CMakeOutput.log".
See also "/data/reserve/input-methond/librime/build/CMakeFiles/CMakeError.log".
Makefile:68: recipe for target 'release' failed

host platfrom :ubuntu18.04

**Comment by ksqsf** [comment, marked as answer]
The error message suggests that you didn't have opencc installed, which is a dependency. You can install it using the package manager or alternatively `make deps`.

**Comment by ruijitek** [comment]
hello ,
     Thank you for your replay！
I already install opencc lib through sudo apt-get install opencc ,
belows is opencc message throug dpkg -l | grep opencc,
![image](https://github.com/rime/home/assets/19148548/4f9c241d-918c-4ceb-b92f-586a699e34c5)

**Reply by ksqsf** [reply]
I don't know which distro you are using, but have you tried to find packages like `libopencc2-dev`?

**Comment by ruijitek** [comment]
I build host distro is ubuntu18.04, kernel version is Linux version 5.4.0-150-generic,
libopencc2-dev packages is not find.

**Reply by fxliang** [reply]
ubuntu18.04 maybe too old...
